### PR TITLE
add ADR for diversity data storage

### PR DESCRIPTION
**Pull request by aldavidson**
## Context

In relation to the Trello card [Store Diversity & Inclusion data securely](https://trello.com/c/l7tYd4AP/483-store-diversity-inclusion-data-securely) we had some fairly involved discussions with @aldavidson, @duncanjbrown, @tijmenb and @bruntonspall about the most appropriate solution. The reasoning, as well as the decision, should be recorded for future reference.

## Changes proposed in this pull request

Add adr/0011-storing-diversity-data.md

## Guidance to review

Do the decision and the reasoning behind it make sense?

## Link to Trello card

[Store Diversity & Inclusion data securely](https://trello.com/c/l7tYd4AP/483-store-diversity-inclusion-data-securely)

## Things to check

N/A

### CODE REVIEW

File: adr/0011-storing-diversity-data.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# 10. Storing Diversity Data
```
Comment (tijmenb) [inline adr/0011-storing-diversity-data.md]:
```suggestion
# 11. Storing Diversity Data
```

File: adr/0011-storing-diversity-data.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+## Context
+
+This service will need to store sensitive diversity data for each candidate - gender, ethnicity, etc - which, although of little interest to attackers, would be potentially impactful to the candidate if someone was to gain unauthorised access to it. Therefore we need to take extra care to minimise the risk of this happening.
+
+## Findings
+
+The risks and threats we considered:
+
+1. Non-malicious but still unauthorised access
+for instance, a genuine provider or support user just clicking around out of interest, or to check up on a family member, etc
```
Comment (duncanjbrown) [inline adr/0011-storing-diversity-data.md]: The formatting makes this read a bit oddly here — github concatenates the lines in the `<ol>`. Full stop?

**Comment by duncanjbrown** [comment]
@aldavidson are you happy for us to fix this up and merge it?

**Comment by aldavidson** [comment]
Yes go ahead!

**Pull request merged**